config/minimalist-templates/button_card_templates.yaml

```diff
@@ -1048,6 +1054,8 @@ card_person:
               return "stretch stretch";
             }
           ]]]
+    label:
+      - text-transform: "capitalize"
```

Comment:
Thank you, that is a really very good idea to make the labels of the person cards translatable. I just don't know yet whether I like the idea that the label is now printed in capital letters. It doesn't really fit in with the way it is displayed everywhere else. What do the rest of you think?